lessons/index.html

```diff
@@ -26,29 +26,54 @@
 	</a>
       </div>
       <div class="span10 offset1">
-	<h1 id="the-shell">The Shell</h1>
-	Author: Sheldon McKay
+	<h1 id="the-shell">Introduction to the command line</h1>
+    <p>Data Carpentry's aim is to teach researchers basic concepts, skills, and tools for working with data so
+    that they can get more done in less time, and with less pain. The lessons below were designed for those
+    interested in working with genomics data through the command line.</p>
+    
+	<p><strong>Author: Sheldon McKay</strong></p>
 	
-	Adapted from the lesson by Tracy Teal. Original contributors: Paul Wilson, Milad Fatenejad, Sasha Wood and Radhika Khetani for Software Carpentry (http://software-carpentry.org/)
+	<p><strong>Adapted from the lesson by Tracy Teal.</strong></p>
+    
+    <p><strong>Original contributors: Paul Wilson, Milad Fatenejad, Sasha Wood and Radhika Khetani for <a href="http://software-carpentry.org">Software Carpentry</a></strong></p>
+    
+    <p><strong>Lesson status: In development</strong></p>
+    
+<!--    <h2 id="instructors">Instructors</h2>
+    See Instructor notes on how to teach this lesson. 
+ -->
 	
-	<h2 id="learning-objectives">Learning Objectives</h2>
```

Comment:
why are we removing the learning objectives? are these being put somewhere else?